src/app/globals.css

```
@@ -1,111 +1,202 @@
 @import "tailwindcss";
 @import "tw-animate-css";
 
-@custom-variant dark (&:is(.dark *));
+/* `class="dark"` en <html> activa modificadores `dark:*` de Tailwind.
+   Necesario porque el default de Tailwind v4 mapea `dark:` a `prefers-color-scheme`,
+   no a una clase ancestor. Mientras los componentes shadcn aún tengan modificadores
+   `dark:*`, mantener este variant. */
+@custom-variant dark (&:where(.dark, .dark *));
```

Stylelint (17.11.0)

[error] Unexpected unknown at-rule "@custom-variant" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)